### ISSUE TITLE
Nexus.tune.createScale() does not produce expected scale

### ISSUE THREAD

**Issue by botway** [nexus-js/ui#138]
Looks like the scale is not played back properly:
``` javascript
// Create a minor scale
Nexus.tune.createScale(0,2,3,5,7,8,10);

// Playing back steps (0,1,2,3,4,5,6,7,8,9,10,11,12)
console.log(step , Tone.Frequency(Nexus.note(step)).toNote());
```
Produces this result:
0 "C4"
1 "D4"
2 "D#4"
3 "F4"
4 "G4"
5 "G#4"
6 "C5"
7 "D5"
8 "D#5"
9 "F5"
10 "G5"
11 "G#5"
12 "C6"

Looks like it jumps to the beginning of the scale after the 6th step. 

Logging Nexus.tune (minor scale above):
scale: Array(6)
0: 1
1: 1.122462048309373
2: 1.1892071150027212
3: 1.3348398541700344
4: 1.4983070768766815
5: 1.5874010519681996
length: 6

Array.length is just 6, that's why it jumps to the beginning.
I am not sure about this, but in tuning.js
```javascript
 createScale() {
    let newScale = [];
    for (let i=0;i<arguments.length;i++) {
      newScale.push( math.mtof( 60 + arguments[i] ) );
    }
    this.loadScaleFromFrequencies(newScale);
  }
// maybe it should be 
 for (let i=0;i<=arguments.length;i++) {...}
```

**Comment by taylorbf** [comment]
Hi. thanks and sorry it took a while to get back to you.

Yes this is a bug. It is actually in the following function, a few lines down from the code you were looking at:

```js
loadScaleFromFrequencies(freqs) {
    this.scale = [];
    for (let i=0;i<freqs.length-1;i++) {
      this.scale.push(freqs[i]/freqs[0]);
    }
  }
```

The loop counts up to `freqs.length-1`, and it should count to `freqs.length`  . This error is due to this function being legacy code from Tune.js, which uses scale data that includes the octave note duplicated at the end of the scale (i.e. scale data that would look like: C D E F G A B C ).

I will aim to revise this in the next round of updates.

**Comment by vladansaracpv** [comment]
Created [pr](https://github.com/nexus-js/ui/pull/179) with changes @taylorbf suggested

**Comment by tatecarson** [comment]
Fixed with #179 .
